### PR TITLE
Remove global router params from docs

### PR DESCRIPTION
**Pull request by rmars**
Global router params were removed as part of #113.  Update documentation accordingly.

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -40,7 +42,6 @@ routers:
       commonName: foo
       caCertPath: /foo/caCert.pem
   timeoutMs: 1000
-  dstPrefix: /ext/http
```
Comment (siggy) [inline docs/config.md]: maybe put this back at the same level as `baseDtab` in the `ext-http` server, and then add `/ext/http => /host/web;` to it's dtab. i think this demonstrates nicely the distinction between an "edge" linkerd and an internal one.

**Comment by siggy** [comment]
thanks! :+1: :star:

File: docs/config.md (hunk 1 above)
Comment (klingerf) [inline docs/config.md]: Ok, really sorry to :bike: :house: on this branch, but I realized that this example config is pretty far out of date.  It doesn't include a namers section, and it has the old, unsupported way of invoking the filesystem namer, plus a few other misc errors that I think we should fix.  How do you feel about these additional changes?

```
diff --git a/docs/config.md b/docs/config.md
index e6814ef..4466ce2 100644
--- a/docs/config.md
+++ b/docs/config.md
@@ -15,23 +15,29 @@ below.
 admin:
   port: 9990
 
+namers:
+- kind: io.l5d.fs
+  rootDir: disco
+
 routers:
 - protocol: http
   label: int-http
   baseDtab: |
-    /host     => /$/io.l5d.fs;
+    /host     => /io.l5d.fs;
     /method   => /$/io.buoyant.http.anyMethodPfx/host;
     /http/1.1 => /method;
   httpUriInDst: true
+  servers:
+  - port: 4140
+    ip: 0.0.0.0
 
 - protocol: http
   label: ext-http
   dstPrefix: /ext/http
   baseDtab: |
-    /host     => /$/io.l5d.fs;
-    /method   => /$/io.buoyant.http.anyMethodPfx/host;
-    /http/1.1 => /method;
-    /ext/http => /host/web;
+    /host         => /$/io.buoyant.http.anyHostPfx/io.l5d.fs/web;
+    /method       => /$/io.buoyant.http.anyMethodPfx/host;
+    /ext/http/1.1 => /method;
   servers:
   - port: 8080
     ip: 0.0.0.0
@@ -54,7 +60,7 @@ routers:
     thriftFramed: true
   thriftMethodInDst: false
   baseDtab: |
-    /thrift => /$/io.l5d.fs/thrift;
+    /thrift => /io.l5d.fs/thrift;
```

Comment (rmars) [inline docs/config.md]: ooh, thanks for catching. let me update.

**Comment by klingerf** [comment]
:star: :bicyclist: Looks great! Thank you for updating.

**Comment by siggy** [comment]
:star: :+1:

**Pull request merged**